docs/user_guide/storing/index_weaviate.md

```diff
@@ -379,8 +380,8 @@ This will perform a hybrid search for the word "hello" and the vector [1, 2] and
 q = (
     store.build_query()
     .text_search(
-        "world", search_field=None  # Set as None as it is required but has no effect
-    )
+        "world"
+    )  # No need to set search_field as it has no effect in hybird search
```

Comment:
I think there is no need to add this annotation here. I as a user I would not see why you mention something that is not there